modules/topeft.py

```diff
@@ -11,13 +11,14 @@
 from coffea.util import load, save
 from optparse import OptionParser
 
+from modules.objects import *
+from modules.corrections import *
```

Comment:
It seems that nothing from corrections.py actually ends up getting used here, should we keep it as a placeholder in case we will want something like it in the future, or possibly just consider removing it for now?

Reply:
In principle corrections.py will be used for unpacking and storing scale factors into .coffea files, that will be open in topeft but probably won't be needed to be imported.